mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -279,9 +280,11 @@ public void setupUserStatusToggle() {
             public void onClick(View v) {
                 if (PrefManager.getUserStatus() == Constants.USER_OFFLINE) {
                     PrefManager.setUserStatus(Constants.USER_ONLINE);
+                    Toaster.show(mNavigationHeader, R.string.online_mode, Toaster.SHORT);
                     userStatusToggle.setChecked(false);
                 } else {
                     PrefManager.setUserStatus(Constants.USER_OFFLINE);
+                    Toaster.show(mNavigationHeader, R.string.offline_mode, Toaster.SHORT);
```

Comment:
I think this message is kind of redundant because the change of switch's color already indicates that we are in Online or Offline mode. But it would be much better if you change the switch's title like "Offline mode " when it's `off` and "Online Mode" when it is `on` or may you can add a summary as we do in preferences.

Reply:
Ok I will do that.

Reply:
Done